### DISCUSSION TITLE
How do I make search work?

### DISCUSSION THREAD

**Discussion by genebean** [thuliteio/doks#896] [Q&A]
I’ve looked at lots of posts here and tried to follow what they suggest beyond what’s in the docs, yet nothing is returned when I type in the box. My test site is [here](https://deploy-preview-124--isd-atl-bsa.netlify.app/) and it’s hooked to [this branch](https://github.com/genebean/indiansprings-hugo/tree/isd-v2).

**Comment by cdiver** [comment]
Copy the index.js file to the assets\js folder - you can grab the index.js file using chrome dev tools. You then can add your own search text to the search index in index.js file
